### PR TITLE
removed deprecated gulp-util

### PR DESCRIPTION
**Pull request by zbennett10**
This pr replaces the recently deprecated gulp-util package with corresponding dependencies. 

For more info see: https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

Thanks!

### CODE REVIEW

**Comment by scniro** [comment]
@zbennett10 [3.9.1](https://www.npmjs.com/package/gulp-clean-css) has been published to npm. Thank you for being on top of these things!

**Comment by zbennett10** [comment]
@scniro you're welcome! Thank you for the tool!

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
   ],
   "dependencies": {
     "clean-css": "4.1.9",
-    "gulp-util": "3.0.8",
+    "fancy-log": "^1.3.2",
```
Comment (phated) [inline package.json]: This should probably be a devDep (only used in gulpfile.js)

Comment (zbennett10) [reply package.json]: Indeed! Good catch.

Comment (phated) [reply package.json]: We want to be careful about where dependencies go so plugin install size can be reduced as much as possible.

Comment (scniro) [reply package.json]: @phated Nice catch, now that `plugin-error` lives on it's own I can definitely fix as you suggested. Will be incoming soon...

Comment (scniro) [reply package.json]: fixed with [3.9.2](https://github.com/scniro/gulp-clean-css/releases/tag/3.9.2)

**Pull request merged**